alertmanager_matrix.go

```diff
@@ -108,15 +123,16 @@ func main() {
 	if err != nil {
 		log.Fatalf("Error connecting to Matrix: %s", err)
 	}
-
+	log.Printf("Connected to Matrix homeserver at %s as %s, and to Alertmanager at %s -- Client instance %+v", homeserver, userID, alertmanagerURL, client)
 	// Start syncing
 	go func() {
 		log.Fatal(client.Run())
 	}()
 
 	// Create/start HTTP server
 	r := mux.NewRouter()
-	r.HandleFunc("/{room}", handler).Methods("POST")
+	bc := &clientHandler{client}
```

Comment:
I don't think this change is needed if this assignment is fixed:

```
client, err := bot.NewClient(homeserver, userID, token, messageType, rooms, alertmanagerURL)
```

(it should be `=` not `:=`)

Reply:
Nono, that wouldn't work.  The client variable being global (and that causing issues in turn) is primarily the central fix I wanted to submit.  As you can see, the variable is no longer global.